bin/wg-meta

```diff
@@ -59,6 +59,9 @@ There are a few honored environment variables L<Wireguard::WGmeta::Index/ENVIRON
 
 =head1 AUTHORS
 
+Since theres a lot of spam flooding my mailbox, I have a quite strict spam filtering in place. If you want to make sure
+that your email gets delivered into my mailbox, include C<#im_not_a_bot#> in the B<subject!>
```

Comment:
```suggestion
Since there is a lot of spam flooding my mailbox, I had to put spam filtering in place. If you want to make sure
that your email gets delivered into my mailbox, include C<#im_not_a_bot#> in the B<subject!>
```